app/containers/Login/LoginPopup.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
Let' rename to LoginOptions. This is not a popup but one of the steps of the popup.